examples/react/getting-started/src/App.tsx

```diff
@@ -69,6 +70,11 @@ export function App() {
               </div>
             </div>
           </div>
+
+          <Chat
+            agentId="8f80fec3-f86a-1c8-86bb-426cb067cbbd"
+            itemComponent={ItemComponent}
```

Comment:
Should we also have a dummy tool to showcase that API?

Reply:
I agree. I'm just not sure if we need to clone the agent in question to update its configuration, or if we can update it directly.
cc @PLNech

Reply:
Ideally this is _our_ agent we can play with and use as a demo?

Reply:
Hey, yes you should create your own! This existing agent is my algobot demo and might change prompts, datasources, etc at any time, I don't guarantee I'll keep it in the current shape at all :stuck_out_tongue: